scripts/write-livecd.sh

```diff
@@ -34,7 +34,7 @@ dev_size=0
 
 # URL or path for ISO. Defaults to latest ISO build on master branch
 
-iso_uri="http://$car/artifactory/csm/MTL/sle15_sp2_ncn/x86_64/dev/master/metal-team/cray-pre-install-toolkit-latest.iso"
+iso_uri="https://artifactory.algol60.net/artifactory/csm-images/stable/cray-pre-install-toolkit/1.5.8/cray-pre-install-toolkit-latest.iso"
```

Comment:
blech.  this breaks when we bump the pit version.

Reply:
Yup, but it at least works. 1.5.8 is a stable iso, vs. the bad link we have already.

Need to publish a latest folder, or publish latest _up_ a folder (so the three latest artifacts are neighboring the versioned subdirs).

Reply:
I merged this into the cray-pre-install-toolkit and then fixed this line up to omit the version.
https://github.com/Cray-HPE/cray-pre-install-toolkit/pull/29